### DISCUSSION TITLE
Inappropriate JA localization "Release date"→「発売日」

### DISCUSSION THREAD

**Discussion by szc126** [VocaDB/community#167] [Ideas]
発売日 refers to the day when something begins to be sold, which is inappropriate for a majority of UTAU characters, etc.

I believe that 配布開始日 "day when distribution began" would be more appropriate, but I want feedback from a native Japanese speaker. But perhaps this wording is odd when applied to commercial entities like VOCALOID.

**Comment by andreoda** [comment]
For UTAUs, since they are usually free to use (fanmade), "配布開始日" would be better. For all other synth programs like Vocaloid, "発売日" fits, since it describes the day when the official selling began.

Maybe "配信日" can be used but I am not sure.

**Reply by szc126** [reply]
`配信日` seems to be for digital distribution of things like music and video games, rarely applied to UTAU and VOCALOID.
